pages/follow-list/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import FollowListBody from 'components/followList/FollowListBody';
+
+function FollowList() {
+  return (
+    <div>
+      <FollowListBody />
+    </div>
```

Comment:
```suggestion
      <FollowListBody />
```
하위가 하나라서 이렇게만 해줘도 괜찮을고야!